scripts/mods/mod_reforged/item_tables/named_northern_weapon_melee.nut

```diff
@@ -0,0 +1,20 @@
+::Reforged.ItemTable.NamedNorthernWeaponMelee <- ::ItemTables.Class.ItemTable([
```

Comment:
How about creating a non-named variant of this same table i.e. `NorthernWeaponMeleeTierMax` and use that?